### PR TITLE
Gives ERT Medical Officers a box of plushies

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
When someone has gone through a stressful or traumatic event, little comforts can go a long way in helping calm them down and speed their recovery. To that end, some emergency services and humanitarian organizations like to carry around teddy bears to give to people after a crisis, [like the Red Cross talks about here](https://www.redcross.org.au/get-involved/connect/volunteer/knitting-for-kindness-trauma-teddies).

So this PR gives the medical officers a box of plushies to hand out to people on station when they're not busy crossing medigun beams or wandering off and getting killed silently in maint. I think it's a nice little way to reinforce that the ERT's are there to help the crew, and give them a little more to do.

## Why It's Good For The Game
Gives medical ERT's something to do when they're not holding down LMB to heal people, plus it gives them a prompt to engage with the crew more than as a free heals dispenser.

## Changelog
:cl: Ryll/Shaps
add: ERT Medical Officers now come equipped with a box of plushies to give to crewmembers in crisis
/:cl:

### CODE REVIEW

**Review by Dafium — COMMENTED** [review]
The code seems legit. I'm still learning but from what i'm seeing, nothing to worry about.

**Comment by Pumpkinoe** [comment]
this is a very important feature please add ASAP

**Comment by KathrinBailey** [comment]
Absolutely brilliant, now we just need headmin ruling to be able to valid anyone who throws the teddies away instead of cherishing them.

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -848,6 +848,27 @@
...
+		/obj/item/toy/plush/snakeplushie,
+		/obj/item/toy/plush/plasmamanplushie,
+		/obj/item/toy/plush/beeplushie,
+		/obj/item/toy/plush/moth,
+		/obj/item/toy/plush/pkplush)
+
+/obj/item/storage/box/hug/plushes/PopulateContents()
+	for(var/i in 1 to 7)
+		var/plush_path = pick(approved_by_corporate)
+		new plush_path(src)
```
Comment (coiax) [inline code/game/objects/items/storage/boxes.dm:870]: Real talk, this is a pet peeve of mine. If you take a plushie out of the box, will it fit back in the box?

Boxes that contain more than they can normally hold bug me.

**Review by coiax — COMMENTED** [review]
Looks good, just have a question on one thing.

File: code/game/objects/items/storage/boxes.dm (hunk 1 above)
Comment (Ryll-Ryll) [reply code/game/objects/items/storage/boxes.dm:870]: Yes you can fit 7 small items in a box

Comment (Mothblocks) [reply code/game/objects/items/storage/boxes.dm:870]: make a unit test for it and fix the stuff that doesnt fit :D

**Pull request merged**